packages/cli/src/__tests__/typeDefsToImplicitResolvers.test.ts

```diff
@@ -52,4 +52,25 @@ describe(supermassive, () => {
       ).toMatchSnapshot();
     }
   });
+
+  it("should generate interfaces with --mandatory-root-operation-types --generate-resolver-map", async () => {
```

Comment:
Why do we have it in this file :D 

Let's just rename it to supermassive.test.ts?